webapp/src/components/shareBoard/shareBoard.tsx

```diff
@@ -135,6 +141,17 @@ export default function ShareBoardDialog(props: Props): JSX.Element {
         await loadData()
     }
 
+    const onLinkBoard = async (channel: Channel, confirmed?: boolean) => {
+        if (channel.type === 'O' && !confirmed) {
```

Comment:
We have a constant for this...https://github.com/mattermost/focalboard/blob/main/webapp/src/blocks/board.ts#L11

Reply:
Actually that is not exactly that, that are "board type" not channel type, I going to create a couple of constants for that in the store/channels.ts because is there where the Channel struct is defined.